src/components/content/Questions.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div>
+    <div v-for="question in questions" :key="question.id">
+      <div v-html="question.body" />
+      <hr>
+    </div>
+  </div>
+</template>
+
+<script>
+  import {mapActions, mapGetters} from 'vuex'
```

Comment:
please verify code style:
```
import { mapActions, mapGetters } from 'vuex'
```

Reply:
Could you check why this rule is not applies?
Link: https://eslint.org/docs/rules/key-spacing

Reply:
Fixed. 
Looks like it is caused by "Optimize imports" feature.